src/modules/auth/component/Login/Login.tsx

```diff
@@ -76,41 +68,56 @@ export function Login() {
       >
         <Grid container spacing={2}>
           <Grid item sm={12}>
-
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit(onSubmit)}>
               <Grid container spacing={6}>
                 <Grid item sm={12}>
-                  <TextField
-                    label="Email"
-                    placeholder="email@gmail.com"
+                  <Controller
                     name="email"
-                    type="email"
-                    size="small"
-                    fullWidth
-                    value={userData.email}
-                    onChange={handleChange}
+                    control={control}
+                    rules={{ required: "Email is required" }}
+                    render={({ field }) => (
+                      <TextField
+                        label="Email"
+                        placeholder="email@gmail.com"
+                        type="email"
+                        size="small"
+                        fullWidth
+                        {...field}
+                      />
+                    )}
                   />
                 </Grid>
                 <Grid item sm={12}>
-                  <TextField
-                    label="Password"
-                    placeholder="********"
+                  <Controller
```

Comment:
Create a PasswordField component. It can be used in sign in, sign up and profile update.
Inside of it, just use newly created TextField from the previous comment and just pass the props to make it specific to the password, like placeholder, type, rules

Reply:
done